apps/playground/src/composables/useStore.ts

```diff
@@ -53,6 +53,8 @@ export const useStore = () => {
         mergeImportMap(importMap.value, {
           imports: {
             "sit-onyx": `https://cdn.jsdelivr.net/npm/sit-onyx@${onyxVersion.value}/dist/index.js`,
+            "@sit-onyx/icons": `https://cdn.jsdelivr.net/npm/@sit-onyx/icons@beta/dist/index.js`,
```

Comment:
We could also add the flags lib here so users can also use flags in the Playground